styles/index.css

```diff
@@ -12,7 +12,181 @@ body {
   background-color: #e5e5e5;
```

Comment:
- [ ] The background color of the body in the figma template is white. Kindly change this to white. You are expected to stick to design template as much as possible, this includes colors, background colors, fonts, sizes, margins and paddings.